ParisBNF/abb/BNFabb202.xml

```diff
@@ -268,12 +274,31 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
 
                                   <item xml:id="a5">
                                     <locus target="#2r"/>
-                                     <desc type="PoemQene"/>
-                                       <q xml:lang="gez"></q>
-                                       <q xml:lang="en">For which generation among the number of generations,
+                                     <desc type="PoemQene">መወደስ፡ ዘ<persName ref="PRS13090TebabaKrestos">ጥበበ፡ ክርስቶስ</persName>።</desc>
+                                       <q xml:lang="gez">
+                                         <l n="1">ለአይ፡ ትውልድ፡ እምኍልቈ፡ ትውልድ።</l>
+                                         <l n="2">ወለማእዜ፡ ዘመን፡ እምአዝማነ፡ አሃዱ።</l>
+                                         <l n="3">ማርያም፡ ለአብ፡ ታቦተ፡ ዋህድ፡ ወልዱ።</l>
+                                         <l n="4">ኪያኪ፡ መድሃኒተ፡ እንዘ፡ የሐስሱ፡ ነቢያት፡ እዱ።</l>
+                                         <l n="5">ቦእለ፡ ተሰዱ፡ እምሀገር፡ ሐገረ፡ ይብጽሖሙ፡ ዘኢለመዱ።</l>
+                                         <l n="6">መከራ፡ ነደ፡ እሳት፡ በነበልባሉ፡ ወነዱ።</l>
+                                         <l n="7">እስከኑ፡ አውአዮሙ፡ ወበልኦሙ፡ ሐበ፡ ከመ፡ እጸው፡ ተአለዱ።</l>
+                                         <l n="8">ኪያከሰ፡ ታቦተ፡ መድሃኒቶሙ፡ ዘክህዱ።</l>
+                                         <l n="9">ህማመ፡ ለርእሶሙ፡ በርእሶሙ፡ ፈቀዱ።</l>
+                                                                         </q>
+                                       <q xml:lang="en">For what generation among the generations,
+                                         For what time among the times,
+                                         Mary, tabot of the only Son of the Father,
+                                         The hand of the prophets were studying you, the salvation,
+                                         Some were exiled from the land, reaching a land they were not accustomed to,
+                                         Examination of the burning fire through its flame and fire,
+                                         Untill it burned them and consumed them like collected wood,
+                                         Them, those who denied you, the tabot of their salvation,
+                                        Illness they caused for themselves though their heads.
```

Comment:
I would understand it rather as "by themselves"

Reply:
Ok, I modified. I think "ay" and "maeze" should be understood in this special way (possible for qene).